docs/application/web/guides/connectivity/download.md

```diff
@@ -85,25 +85,45 @@ To provide the user access to Internet resources, you must learn how to manage d
 
    The `start()` method returns a unique identifier for the download operation.
 
-5. During the download:
+   You can check the status of the download operation by calling the getState() method with the download ID:
+
+      ```
+      tizen.download.getState(downloadId);
+      ```
+
+5. During the download operation, you can modify the download state.
+
+    >**Note**
+    >
+    >Download operation can become completed almost instantaneously, depending on download size and connection speed.
 
    1. To pause the download, use the `pause()` method with the download ID:
 
       ```
       tizen.download.pause(downloadId);
       ```
+        Calling the pause() method will result in the PAUSED download state only if the previous state was DOWNLOADING or QUEUED.
 
    2. To resume the download, use the `resume()` method with the download ID:
 
       ```
       tizen.download.resume(downloadId);
       ```
+        Calling the resume() method will result in the DOWNLOADING or the QUEUED download state only if the previous state was PAUSED, CANCELLED or FAILED.
 
    3. To cancel the download, use the `cancel()` method with the download ID:
 
       ```
       tizen.download.cancel(downloadId);
       ```
+        Calling the cancel() method will result in the CANCELED download state only if the previous state was DOWNLOADING or QUEUED.
```

Comment:
@apietraszek 

Request you to change:   
CANCELED --> CANCELLED

Reply:
I have changed others to CANCELED, that is the proper enumerator value.

Reply:
@apietraszek 

thanks for the clarification, 
Line 112, please change it to CANCELED

Reply:
It's CANCELED in whole file now.